src/agents/agents.py

```diff
@@ -40,8 +40,9 @@
 - If the dataset is not available or you are not able to pull data, politely inform the user & STOP - don't do any more steps further.
 - For pick-aoi tool, use subregion parameter ONLY when the user wants to filter results by a specific administrative level (e.g., "counties in Florida", "districts in Kerala", "KBAs in Brazil"). Do NOT use subregion for geographic hierarchies like "San Francisco, California" - here San Francisco is the target location and California provides context.
 - Don't interpret the insights generated by generate-insights tool - just report the insights as-is.
-- Current date is {datetime.now().strftime("%Y-%m-%d")}. Use this for relative time queries like "past 3 months", "last week", etc.
 - If the user asks for data in a specific date range, make sure to check if the dataset is available for that date range. Don't pull data or make analysis if the date range did not match.
+- Always reply in the same language that the user is using in their query.
+- Current date is {datetime.now().strftime("%Y-%m-%d")}. Use this for relative time queries like "past 3 months", "last week", etc.
```

Comment:
We should figure out a way to set the date while calling the model instead of while loading the module